test/JDBC/expected/database_metadata.out

```diff
@@ -0,0 +1,276 @@
+
+create table dbmeta_tab1(col1 int primary key, col2 nvarchar(max) not null)
+create table dbmeta_tab2(fcol1 int not null, foreign key (fcol1) references dbmeta_tab1(col1))
+create function dbmeta_func1() returns int as begin return 1 end
+create function dbmeta_func2(@par1 int) returns table return select cast(@par1 as int) as col1, cast ('foo' as nvarchar(max)) as col2
+create index idx1 on dbmeta_tab1(col2)
+create unique index idx2 on dbmeta_tab1(col2)
+create procedure dbmeta_proc1 as begin return end
+create procedure dbmeta_proc2 @par1 int as select cast(@par1 as int) as col1, cast ('foo' as nvarchar(max)) as col2
+
+dbmeta#!#getCatalogs
+~~START~~
+varchar
+master
+msdb
+tempdb
+~~END~~
+
+
+dbmeta#!#getColumnPrivileges#!#master|dbo|dbmeta_tab1|col1
+~~START~~
+varchar#!#varchar#!#varchar#!#varchar#!#varchar#!#varchar#!#varchar#!#varchar
+master#!#dbo#!#dbmeta_tab1#!#col1#!#dbo#!#dbo#!#INSERT#!#YES
+master#!#dbo#!#dbmeta_tab1#!#col1#!#dbo#!#dbo#!#REFERENCES#!#YES
+master#!#dbo#!#dbmeta_tab1#!#col1#!#dbo#!#dbo#!#SELECT#!#YES
+master#!#dbo#!#dbmeta_tab1#!#col1#!#dbo#!#dbo#!#UPDATE#!#YES
+~~END~~
+
+
+dbmeta#!#getTables#!#master|dbo|dbmeta_%
+~~START~~
+varchar#!#varchar#!#varchar#!#varchar#!#varchar
+master#!#dbo#!#dbmeta_tab1#!#TABLE#!#<NULL>
+master#!#dbo#!#dbmeta_tab2#!#TABLE#!#<NULL>
+~~END~~
+
+dbmeta#!#getTables#!#master|dbo|dbmeta_%|TABLE|VIEW
```

Comment:
is it expected we get no rows?

Reply:
Empty result set was caused by #2404 , it is fixed in latest 4x. Rebased the branch to latest and added corresponding output.